src/screens/Dashboard/Dashboard.js

```diff
@@ -16,6 +16,8 @@ import { DataLarge, H5, H6, Figure, ButtonTertiaryLabel } from '../../components
 import Tooltip from '../../components/Tooltip';
 import Skeleton from '../../components/Skeleton';
 import { MicroSpinner } from '../../components/Spinner';
+import snxJSConnector from '../../helpers/snxJSConnector';
+import { debounce } from 'lodash';
```

Comment:
Hey @bachstatter!

I'm a new dev here at SNX. Thanks for your PR.

```suggestion
import debounce from 'lodash/debounce';
```

@bachstatter could you please put absolute imports on top? Thx.
To keep imports neat its better to group them together, so perhaps put a before the `snxJSConnector` (since its not related to the components group)